spec/std/exception/call_stack_spec.cr

```diff
@@ -57,4 +57,21 @@ describe "Backtrace" do
     output.to_s.empty?.should be_true
     error.to_s.should contain("Invalid memory access")
   end
+
+  pending_win32 "print exception with non-existing PWD" do
+    source_file = datapath("blank_test_file.txt")
+    compile_file(source_file) do |executable_file|
+      output, error = IO::Memory.new, IO::Memory.new
+      with_tempfile("non-existent") do |path|
+        Dir.mkdir path
+        Dir.cd(path) do
+          # on win32 it seems not possible to remove the directory while we're cd'ed into it
```

Comment:
From https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/rmdir-wrmdir?view=msvc-170#remarks:

> The directory must be empty, and it must not be the current working directory or the root directory.

And https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/rmdir:

> You can't use the rmdir command to delete the current directory. If you attempt to delete the current directory, the following error message appears:
>
> `The process can't access the file because it is being used by another process.`
>
> If you receive this error message, you must change to a different directory (not a subdirectory of the current directory), and then try again.